src/test/SDL_test_common.c

```diff
@@ -708,7 +693,6 @@ SDLTest_CommonState *SDLTest_CommonCreateState(char **argv, SDL_InitFlags flags)
     state->window_w = DEFAULT_WINDOW_WIDTH;
     state->window_h = DEFAULT_WINDOW_HEIGHT;
     state->logical_presentation = SDL_LOGICAL_PRESENTATION_DISABLED;
-    state->logical_scale_mode = SDL_SCALEMODE_LINEAR;
```

Comment:
Does this field still exist? I didn't see a header change?

Reply:
Whoa, good catch...removed now in 21bea444bfdf2ae24df49fde03d83f41a3cbfa59!